include/cute/arch/mma_xe.hpp

```diff
@@ -68,6 +68,161 @@ SYCL_DEVICE_OCL(float  intel_sub_group_tf32_tf32_matrix_mad_k8(float a, cute::in
 
 #undef SYCL_DEVICE_OCL
 
+#if defined(__SYCL_DEVICE_ONLY__) && defined(SYCL_INTEL_TARGET)
+#define CUTE_ARCH_COPY_XE_ENABLED
+#endif
+ 
+#if defined(CUTE_ARCH_COPY_XE_ENABLED) && defined(__INTEL_LLVM_COMPILER) && (__INTEL_LLVM_COMPILER < 20250200)
+#define CUTE_ARCH_COPY_XE_BUILTIN_ENABLED
+#elif defined(CUTE_ARCH_COPY_XE_ENABLED)
+#define CUTE_ARCH_COPY_XE_SPIRV_ENABLED
+#endif
+
+namespace cute::detail{
+template <class DElement, class AElement, class BElement, class CElement>
+struct OpSubgroupMatrixMultiplyAccumulateINTEL {};
```

Comment:
Likewise I think we should rename this to disambiguate from the SPIRV name. `XeSubgroupMatrixMultiplyAccumulate` perhaps?